### PR TITLE
A few code and docs bug fix and docs improvement.

### PR DESCRIPTION
**Pull request by ShinoharaYuuyoru**
1. Code
  1.1. lif_fc_mnist.py
    Message output format changed. Please check the tutorial.clock_driven(_en).rst and ~/clock_driven(_en)/3_fc_mnist.rst.
  1.2. tempotron_mnist.py
    1) I found that the original code miss `optimizer.zero_grad()` during trianing. I added it and re-trained the model, finally got about 84% accuracy in the test set. I think this is a bug fix.
    2) Improve the related docs with the new result. However, I didn't translate the chinese docs to the English one because my english is poor... :(

2. Docs
  2.1. Fix some small bugs and warnings although some of them have no effects to the final docs output. :)
  2.2. Improvements have been already mentioned in 1.

Thank you.

### CODE REVIEW

**Comment by fangwei123456** [comment]
https://github.com/fangwei123456/spikingjelly/commit/0c061e49e6d74e79d934306b82074c5f665437e1
